cardano-cli/test/cardano-cli-golden/Test/Golden/CreateTestnetData.hs

```diff
@@ -26,22 +39,16 @@ tree root = do
   subTrees <- mapM tree subs
   return $ files ++ concat subTrees
 
+-- | This test tests the non-transient case, i.e. it maximizes the files
+-- that can be written to disk. Execute this test with:
+-- @cabal test cardano-cli-golden --test-options '-p "/golden create testnet data/'@
```

Comment:
I agree that we should add it somewhere instead of haddock. I think this is the best place: https://github.com/input-output-hk/cardano-node-wiki/wiki/Running-tests 
We can link from `CONTRIBUTING.md` to it then.

Reply:
I've added PR there: https://github.com/input-output-hk/cardano-node-wiki/pull/37